spec/std/int_spec.cr

```diff
@@ -777,4 +777,11 @@ describe "Int" do
     65.unsafe_chr.should eq('A')
     (0x10ffff + 1).unsafe_chr.ord.should eq(0x10ffff + 1)
   end
+
+  it "#bit_length" do
+    0.bit_length.should eq(0)
+    (1..100).each do |i|
+      i.bit_length.should eq(Math.log2(i).to_i + 1)
```

Comment:
This is testing that the result of `bit_length` matches some mathematical formula. But it has about as much to do with the expected answers as the actual implementation of the method. Should you then have to write tests for this mathematical formula?
Tests shouldn't have logic, I'd suggest checking actual inputs.

Reply:
Things like `0b1000` and `0b111.bit_length` might look nice

Reply:
Good idea. Done!

Reply:
Nice, thanks. But I think it still needs some checks specifically with "1+all-zeros" (like `0b1000`) and "all-ones" (`0b111`) to catch off-by-one errors.

Reply:
How about now?